CONTRIBUTING.md

```diff
@@ -0,0 +1,54 @@
+# Contributing to SUI Components
+
+The following is a set of guidelines for contributing with developping new components.
+
+## Getting started
+
+### Environment
+
+To develop new packages, you only need to install Node with NPM. Fin below the default versions this repo was built with:
+* Node: `v6+`
+* NPM: `v3`
+
+### Main commands
+
+This repo is managed with [sui-studio](https://www.npmjs.com/package/@s-ui/studio) tool.
+
+Some base commands (based on sui-studio) you should acknowledge :
+* `npm run generate <category> <name>` to create a new component.
+* `npm run co` to commit your changes with a wizard.
```

Comment:
With a wizard?

![image](https://user-images.githubusercontent.com/1561955/31018528-c7b3448a-a52c-11e7-90b8-e8f21b4d25dc.png)

Reply:
https://en.wikipedia.org/wiki/Wizard_(software) LOL will change.
I also mispelled the examples

Reply:
hahahah don't worry! I know `wizard` is fine is just that I thought in Gandalf while reading it. xD